examples/ex9p.cpp

```diff
@@ -198,11 +213,13 @@ int main(int argc, char *argv[])
    {
       pmesh->UniformRefinement();
    }
+   pmesh->ExchangeFaceNbrData();
 
    // 7. Define the parallel discontinuous DG finite element space on the
    //    parallel refined mesh of the given polynomial order.
-   DG_FECollection fec(order, dim);
+   DG_FECollection fec(order, dim, BasisType::GaussLobatto);
    ParFiniteElementSpace *fes = new ParFiniteElementSpace(pmesh, &fec);
+   fes->ExchangeFaceNbrData();
```

Comment:
The one for the `pmesh` was not necessary, I removed it, however the one for `fes` is necessary (SegFault otherwise).
@v-dobrev Is it okay to add `ExchangeFaceNbrData()` in the constructor of `ParFiniteElementSpace`?

Reply:
I agree that this should be triggered somewhere internally.